flink-end-to-end-tests/test-scripts/container-scripts/docker-compose.test.yml

```diff
@@ -16,15 +16,31 @@
 # limitations under the License.
 ################################################################################
 
-# Extensions to flink-container/docker/docker-compose.yml that mounts volumes needed for tests
+# Docker compose file for a Flink job cluster deployment.
+#
+# Parameters:
+# * FLINK_DOCKER_IMAGE_NAME - Image name to use for the deployment (default: flink-job:latest)
```

Comment:
default seems to be `flink-job`, w/o the `:latest`.

Reply:
:latest is effectively implied.

Reply:
Fair point.

Reply:
This was also copie as is from `flink-container/docker/docker-compose.yml`